.circleci/config.yml

```diff
@@ -29,10 +29,13 @@ jobs:
       - checkout
       - run:
           name: "install requirements"
-          command: "pip install -r requirements.txt"
-      - run:
-          name: "run pytest"
-          command: "pytest"
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -r requirements.txt
+          command: |
+            . venv/bin/activate
+            pytest
```

Comment:
This is not correct!  I made 2 separate suggestions, you can't combine them like this.